app/views/people/_attrs.html.haml

```diff
@@ -0,0 +1,46 @@
+-#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
```

Comment:
When a view partial should be extended inside a wagon, we don't copy the core partial and adjust some small stuff

In general the way to go here is to define a new partial in the wagon and then use `render_extensions` to add the part we want to add

A good example is `_details_sac_cas.html.haml` or `_fields_sac_cas.html.haml`

since you already have `_data_quality_sac_cas.html.haml`, you should be able to call

`render_extensions: :data_quality`

in the core partial and removing this copied file

Reply:
wow, that's clever
https://github.com/hitobito/hitobito/pull/2781